app/src/app.py

```diff
@@ -21,7 +21,7 @@
 def create_app() -> APIFlask:
     app = APIFlask(__name__)
 
-    root_logger = src.logging.init(__package__)
+    root_logger = src.logging.Log(__package__).root_logger
```

Comment:
Probably a better way to do this, I didn't want to do a `with ...` here because we just want it to live forever for the lifespan of an app so don't care about exit being called.

Reply:
Two thoughts. First, if we want to preserve the context manager abstraction and not rely on the fact that there's a class instance variable called root_logger (see my comment below about just using a `@contextmanager` decorator rather than creating the `Log` class), then we can do

```python
root_logger = src.logging.init(__package__).__enter__()
```

But my second thought is that we only care that `src.logging.init()` initializes the logging, we don't _really_ care about the return value, and this is the only place in the code where we use the return value from init anyways, so my instinct is to remove the return value from `src.logging.init` algother, and change this code to just

```
src.logging.init(__package__)
flask_logger.init_app(logging.root, app)
```